content/en/dynamic_instrumentation/symdb/dotnet.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Enable autocomplete and search for .NET
+kind: Documentation
+aliases:
+    - /tracing/dynamic_instrumentation/enabling/dotnet/
+is_beta: false
+private: false
+code_lang: dotnet
+type: multi-code-lang
+code_lang_weight: 30
+further_reading:
+    - link: 'agent'
+      tag: 'Documentation'
+      text: 'Getting Started with Datadog Agent'
```

Comment:
```suggestion
```
You have to include the following at the end of the page for the further reading links to display:

```
## Further reading
 
{{< partial name="whats-next/whats-next.html" >}}
```